src/posts/Engaging with neume.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Engaging with neume
+date: '2022-07-05'
+tags: 
+description: Detailing the preferred channels and processes for engagement with neume 
+permalink: posts/{{ title | slug }}/index.html
+---
+
+# Engaging with neume
+
+neume, as an open-source project, is dependent on the community surrounding it. As the project develops it’s important that we grow and refine the basis on which the community is encouraged to engage.
+
+## Discussion
+
+Currently we have three preferred communication channels:
+- On-going **text chat** within the neume channels in the [HIFI Labs discord](https://discord.gg/P5rrpZN4ds)
+    - Any and all questions around the project can be asked, across #neume-dev (technical) and #neume-chat (general)
```

Comment:
for some reason these indented bullet points are displayed as such on the site :(

Reply:
Hmm good catch. I will play around to see if I can get indented bullets and if not then combine into a single bullet